files/en-us/web/css/scaling_of_svg_backgrounds/index.html

```diff
@@ -26,6 +26,11 @@ <h2 id="The_algorithm_in_summary">The algorithm, in summary</h2>
 
 <p>It's worth noting that the sizing algorithm only cares about the image's dimensions and proportions, or lack thereof. An SVG image with fixed dimensions will be treated just like a raster image of the same size.</p>
 
+<div class="notecard note">
+  <h4>Note</h4>
+  <p>If you are trying to stretch your SVG to a different aspect ratio with CSS&mdash;for example in order to stretch it over the page background&mdash;make sure your SVG includes <code>preserveAspectRatio="none"</code>. Find out more about {{svgattr("preserveAspectRatio")}}.</p>
```

Comment:
```suggestion
  <p>If you are trying to stretch your SVG to a different aspect ratio with CSS&mdash;for example in order to stretch it over the page background&mdash;find out more about {{svgattr("preserveAspectRatio")}}, and make sure your SVG includes <code>preserveAspectRatio="none"</code>.</p>
```
This is just nit but to me at least it seems to read a bit more naturally this way.

Reply:
I think the key bit of advice is to include preserveAspectRatio rather than go read about it however, so I'd kind of rather keep it the way it is. I'd really just added the link as we have it for anyone who might be wondering why.